contrib/storage-hive/core/pom.xml

```diff
@@ -142,6 +142,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Where is the dependency introduced?

Reply:
OK, I see. I was confused due to Drill practice to use the same package name in different modules/jars. This practice is not common and far from best practices.

Reply:
Could you clarify what package name would be better to use?
Do you mean short names instead of full ones for the common packages?

Reply:
No, I don't refer to using short names. I refer to using "org.apache.drill.exec" package name outside of "exec". As it is already widely used in Drill I don't suggest to change it in this PR.